pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1139,6 +1140,18 @@ public void setExistingTableConfig(AbstractTableConfig config, String tableNameW
     } else if (type == TableType.OFFLINE) {
```

Comment:
For HLC, no because we have only one replica per segment.

For LLC, I tried to make a change but then backed it out as per comments, because a much bigger change is needed to increase number of replicas.